src/types/upload/index.ts

```diff
@@ -130,3 +130,8 @@ export interface UploadFile extends BackupedFile {
     encryptedKey: string;
     keyDecryptionNonce: string;
 }
+
+export interface ParsedExtractedMetadata {
+    location: Location;
+    creationTime: number;
```

Comment:
Should we mark these fields as nullable/optional?
location`?`,
creationTime`?`

Also, instead of  `ParsedExtractMetadata`, maybe we can rename it as `ExtractedVideoMetadata`?

Reply:
1. There were two approaches 

either I let them be optional and use `?.` whenever accessing them to check to avoid accessing a property when the value is undefined 

or make sure that they are never undefined and return a null value when let's say we don't get the data 
this way I can avoid the conditional accessing `?.` 

I have taken the second route as lit looked  cleaner

2.  `ParsedExtractMetadata` type is used for both the metadata extracted from EXIF and video metadata